rxv/rxv.py

```diff
@@ -89,7 +89,8 @@ class RXV(object):
 
     def __init__(self, ctrl_url, model_name="Unknown",
                  serial_number=None, zone="Main_Zone",
-                 friendly_name='Unknown', unit_desc_url=None):
+                 friendly_name='Unknown', unit_desc_url=None,
+                 http_timeout=60.0):
```

Comment:
I think we should lower the default value a bit.
IIRC home assistant polls rxv quite often (every 10s?) 
Maybe we should match this?

In my experience Yamahas respond rather quickly

Reply:
That's been my experience as well. 10s sounds good to me.